src/e2e-node/README.md

```diff
@@ -2,8 +2,10 @@
 
 ## Running the tests
 
-To run the tests, run `npm install` at the root. Then, copy
-`/src/e2e-node/.env.defaults` into `/src/e2e-node/.env.test.local`,
-and insert the credentials of a Solid Pod to use for testing.
+To run the tests locally:
 
-You can now run `npm run e2e-tests-node` at the root.
+1. At the root, run `npm install`.
+2. Copy `.env.default` to `.env.test.local`.
```

Comment:
I the last commit to my PR, I renamed `env.default(s)` to be `.env.example` (#9a0091e) - the commit message was:

Better reflects the purpose of these files, and avoids potential
confusion over using as fallback config: https://www.npmjs.com/package/dotenv-flow#alternative-defaults-envdefaults

Reply:
I thought it made sense to have it automatically fall back to values for e.g. the server URL, but no strong opinion there: 26fccf7e

Reply:
Ah and 10bab4a4 of course.